src/pages/PaniniCreator.tsx

```diff
@@ -31,52 +33,91 @@ export default function PaniniCreator(props: PaniniCreatorProps) {
-            <img src="/src/images/wheat.svg" alt="wheatIcon" className={styles.wheatIcon}></img>
-          </SwipeSection>
-          <SelectSection removable={true} title="cheese" options={cheeseVariants}></SelectSection>
-          <SelectSection removable={true} title="meat" options={meatVariants}></SelectSection>
-          <SwipeSection removable={true} title="dressing" options={dressingVariants}></SwipeSection>
-          <CheckboxButtonSection removable={true} title="vegetables" options={vegetableVariant}></CheckboxButtonSection>
-        </article>
-      </Form>
-      <Form title="Configure Extras">
-        <article className={styles.formSections}>
-          <SelectSection removable={true} title="Egg" options={eggVariants}></SelectSection>
-          <CheckboxSection removable={false} title="Egg" options={spreadVariant}></CheckboxSection>
-          <RadioSection removable={false} title="Serving" options={servingVariant} name={"serving"}></RadioSection>
-          <CheckboxSection removable={false} title="Topping" options={toppingVariant}></CheckboxSection>
-        </article>
-      </Form>
-      <Form title="Finalize Order">
-        <article className={styles.formSections}>
-          <TextSection title="Name panini"></TextSection>
-          <CheckboxSection removable={false} title="Cutlery" options={["Add to order"]}></CheckboxSection>
-          <CheckboxSection removable={false} title="Name panini" options={["Add to order"]}></CheckboxSection>
-        </article>
-        <div className={styles.formsSubmitInterfaceWrapper}>
-          <NavLink to={props.navTo} onClick={setOrderDataToTrue}>
-            <label className={styles.formsSubmitLabel}>
-              place order or start again
-              <input type="submit" className={styles.formsSubmit} value={"place order"} />
-            </label>
-          </NavLink>
-          <NavLink to="/panini_creator" onClick={resetOrderData} className={styles.formsResetNavLink}>
-            <button className={styles.formsReset}>start again</button>
-          </NavLink>
+    <FormProvider {...methods}>
+      <form className={styles.paniniCreator} onSubmit={methods.handleSubmit(handleSave)}>
+        <div className={styles.formsInterface}>
+          <h2 className={styles.formsLabel}>Panini Creator</h2>
+          <button type="button" className={styles.button}>
+            <img className={styles.diceIcon} src="/src/images/dices.svg" alt="dices icon"></img>
+            Randomize Panini
+          </button>
         </div>
-      </Form>
-    </main>
+        <FormCard title="Configure Base">
+          <div className={styles.formSections}>
+            <SwipeSection removable={false} name="bread" title="bread" options={breadVariants}>
+              <img src="/src/images/wheat.svg" alt="wheatIcon" className={styles.wheatIcon}></img>
+            </SwipeSection>
+            <SelectSection
+              removable={true}
+              name="cheese"
+              title="cheese"
```

Comment:
**nitpick:** I would keep fields names in some enum, like `enum FormFields` to prevent duplicating strings, and improving maintainability